src/MongoDB/Monitoring/CommandFailedEvent.c

```diff
@@ -245,6 +247,7 @@ static HashTable* php_phongo_commandfailedevent_get_debug_info(zval* object, int
 
 	sprintf(operation_id, "%" PRIu64, intern->operation_id);
 	ADD_ASSOC_STRING(&retval, "operationId", operation_id);
+	PHONGO_BSON_INIT_STATE(reply_state);
```

Comment:
Would it make sense to move this as close as possible to the declaration? It'd certainly be clearer here, but possibly also in methods earlier in the PR (before we assign `intern`). That would minimize any risk of us introduce new code that uses the state without it being properly initialized.